src/app.css

```diff
@@ -2,24 +2,25 @@
 @tailwind components;
 @tailwind utilities;
 
-
-
 @layer components {
+	body {
+		@apply dark:bg-gray bg-white;
+		@apply dark:text-white text-gray;
+	}
 
-  body {
-    @apply bg-gray-light;
-  }
+	a {
```

Comment:
We should not do this in general for all links. For the case you implemnted it it is only for the navigation, isn't it? Because this are special links they maybe should be represented by a function or this should be added to the navbar component